CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+# 3.0.1 - [#21](https://github.com/ServiceInnovationLab/openfisca-aotearoa/pull/21)
+
+* Tax and benefit system evolution. Minor change.
```

Comment:
If we removed exposed variables that's a breaking change 🙈

Reply:
They were replaced in the previous major release (and flagged in the changelog) - and moved to /variables/entitlements/rates_rebates.py.  This file was overlooked and should have been removed. Technically it's a breaking change yes.